kubernetes/linux/main.sh

```diff
@@ -288,6 +288,27 @@ fi
 export CLOUD_ENVIRONMENT=$CLOUD_ENVIRONMENT
 echo "export CLOUD_ENVIRONMENT=$CLOUD_ENVIRONMENT" >> ~/.bashrc
 
+# Copying over CA certs for airgapped clouds. This is needed for Mariner vs Ubuntu hosts.
+# We are unable to tell if the host is Mariner or Ubuntu,
+# so both /anchors/ubuntu and /anchors/mariner are mounted in the yaml.
+# One will have the certs and the other will be empty.
+# These need to be copied to a different location for Mariner vs Ubuntu containers.
+# OS_ID here is the container distro.
+# Adding Mariner now even though the elif will never currently evaluate. 
+if [ $CLOUD_ENVIRONMENT == "usnat" ] || [ $CLOUD_ENVIRONMENT == "ussec" ]; then
+  OS_ID=$(cat /etc/os-release | grep ^ID= | cut -d '=' -f2)
+  if [ $OS_ID == "ubuntu" ]; then
+    cp /anchors/ubuntu/* /usr/local/share/ca-certificates/
+    cp /anchors/mariner/* /usr/local/share/ca-certificates/
```

Comment:
if this an ubuntu system then /anchors/mariner/ will be empty? If so, why this extra copy.  vice-versa in the else clause

Reply:
The OS_ID is for the container distro but the /anchors/ubuntu and anchors/mariner are coming from the node which can have a different distro than the container, such as a Mariner host and our Ubuntu container. We do not know the system of the node so we need to mount both and one directory will always be empty. I could add an if statement to only copy over the directory that isn't empty, but have checked trying to copy both doesn't cause issues

The elif is for when we move over to use mariner as our container distro. When this happens, there can be an Ubuntu host for our Mariner container, and the certs need to be copied over into a different directory than for an ubuntu container